### PR TITLE
treeherder graphs MVP

### PR DESCRIPTION
**Pull request by wlach**
Just looking for feedback, please do not merge this.

### CODE REVIEW

File: webapp/app/js/perf.js [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+                });
+                testInputCreated = true;
+              });
+  };
+
+  $modalInstance.updateTestInput = $scope.updateTestInput;
+});
+
+perf.config(function($stateProvider, $urlRouterProvider) {
+  $stateProvider.state('graphs', {
```
Comment (wlach) [inline webapp/app/js/perf.js]: Done

**Comment by edmorley** [comment]
Presumably this will need gruntfile changes too?

**Comment by wlach** [comment]
Updated the PR to include gruntfile changes (and also rearranged things to use treeherder's standards)

File: webapp/app/js/perf.js [hunk 2]
```diff
@@ -0,0 +1,388 @@
...
+    this.$get = function() {
+        if (window.thServiceDomain) {
+            return window.thServiceDomain;
+        } else {
+            return "";
+        }
+    };
+});
+
+var SERIES_COLORS = [ 'red', 'green', 'blue', 'orange', 'purple' ];
```
Comment (maurodoglio) [inline webapp/app/js/perf.js]: can you wrap it in a value definition? See https://github.com/mozilla/treeherder-ui/blob/master/webapp/app/js/values.js for an example

File: webapp/app/js/perf.js [hunk 3]
```diff
@@ -0,0 +1,388 @@
...
+
+var PLOT_OPTIONS = {
```
Comment (maurodoglio) [inline webapp/app/js/perf.js]: same here

File: webapp/app/js/perf.js [hunk 4]
```diff
@@ -0,0 +1,388 @@
...
+        color: '#cdd6df',
+        borderWidth: 2,
+        backgroundColor: '#fff',
+        hoverable: true,
+        clickable: true,
+        autoHighlight: false
+    }
+};
+
+function getSeriesSummary(signature, signatureProps, projectName, number) {
```
Comment (maurodoglio) [inline webapp/app/js/perf.js]: I would wrap this in a service

**Comment by wlach** [comment]
Updated with requested changes (actually I just put PLOT_OPTIONS inline because they were only used in one place)

**Comment by wlach** [comment]
I made the textbox in the series picker inactive (replaced with some text letting user know what's going on), added as a separate commit.

**Pull request merged**